### ISSUE TITLE
compatibility problem

### ISSUE THREAD

**Issue by peow** [jj21asd/ContainerSearch#3]
There is a compatibility issue with the Inventory Tabs mod, where you can access other chests near you without having to leave the chest you opened, the problem is that the overlays of the 2 mods overlap
![2023-10-04_16 56 38](https://github.com/jjblock21/ContainerSearch/assets/105888751/8fd44561-9dae-410a-a441-291c86c148cc)

I imagine that a simple way to solve this is to give the possibility to change the position of the search bar

**Comment by jjblock21** [comment]
Yes, being able to customize the search bar position was a feature that other people requested, I just procrastinated the update for a month.
I have most of the code ready though and will probably update the mod soon.

**Comment by jjblock21** [comment]
The mod should work with 1.20 and 1.20.1
Not sure if it works with 1.20.2 or 1.20.3 but I don't think there were any critical changes.

**Comment by sternschnaube** [comment]
Any update on this? 😊

**Comment by jjblock21** [comment]
I'm working on it already, probably will release a fix for 1.20.2 tomorrow or on Tuesday and then work on the other features.

**Comment by sternschnaube** [comment]
Awesome!!
